### ISSUE TITLE
Deprecation Warning on RubyVM::DebugInspector

### ISSUE THREAD

**Issue by prpetten** [banister/binding_of_caller#85]
Just started getting this deprecation warning.

````ruby
.rbenv/versions/3.0.5/lib/ruby/gems/3.0.0/gems/binding_of_caller-1.0.0/lib/binding_of_caller/mri.rb:21: warning: constant RubyVM::DebugInspector is deprecated
````

Looking into it a bit, it looks like `debug_inspector` updated its name and usage in a recent release.

https://github.com/banister/debug_inspector/pull/37

**Comment by prpetten** [comment]
Created a PR for this

#86

**Comment by prpetten** [comment]
Released in v 1.0.1
